docs/source/reference/index.rst

```diff
@@ -22,3 +22,4 @@ Reference
    graph
    caffe
    check
+   distributions
```

Comment:
I feel it's better to place this document to some upper place (e.g. bellow `links`). (maybe we need more structure to this reference section, but that is another issue)